zkevm-circuits/src/sig_circuit/test.rs

```diff
@@ -45,7 +45,28 @@ fn sign_verify() {
 
         log::debug!("end of testing for msg_hash = 0");
     }
-    // msg_hash != 0
+    // msg_hash == 1
```

Comment:
how about `msg_hash == 0`? this can also be a possible case in ecrecover. ref #930

Reply:
msg_hash == 0 already added in the sig_circuit/test.rs before